lib/response.js

```diff
@@ -407,8 +407,8 @@ res.sendFile = function sendFile(path, options, callback) {
   var next = req.next;
   var opts = options || {};
 
-  if (!path) {
-    throw new TypeError('path argument is required to res.sendFile');
+  if (typeof path !== 'string') {
+    throw new TypeError('path must be string');
```

Comment:
Also, can you add back `res.sendFile` to the message? Helps people know what function the issue was to.